src/test/java/org/opensearch/knn/integ/PainlessScriptHelper.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.knn.integ;
+
+import org.opensearch.common.xcontent.XContentFactory;
+import org.opensearch.core.xcontent.ToXContent;
+import org.opensearch.core.xcontent.XContentBuilder;
+import org.opensearch.knn.common.KNNConstants;
+import org.opensearch.knn.index.engine.KNNMethodContext;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Objects;
+
+public class PainlessScriptHelper {
+
+    /**
+     * Utility to create a Index Mapping with multiple fields
+     */
+    public static String createMapping(List<MappingProperty> properties) throws IOException {
```

Comment:
I think we should add coding convention of adding final for all parameters just to be consistent across all files. I am not a fan of it but some people want it to be that way and I think having final is where we will go in a long run.

Reply:
updated